pkg/ctl/delete/nodegroup_test.go

```diff
@@ -52,5 +52,14 @@ var _ = Describe("delete", func() {
 			args:  []string{"nodegroup", "ng", "--cluster", "dummy", "--name", "ng"},
 			error: fmt.Errorf("Error: --name=ng and argument ng cannot be used at the same time"),
 		}),
+
+		Entry("setting --parallel below 1", invalidParamsCase{
+			args:  []string{"nodegroup", "--cluster", "dummy", "--name", "ng", "--parallel", "-1"},
```

Comment:
```suggestion
			args:  []string{"nodegroup", "--cluster", "jake", "--name", "ng", "--parallel", "-1"},
```
@Callisto13 doing your job for you! too easy!